### ISSUE TITLE
Tiles showing the wrong color

### ISSUE THREAD

**Issue by TheVekter** [tgstation/tgstation#51281]
Issue reported from  Round ID: 137958 (/tg/Station Manuel [ENGLISH] [US-WEST] [ROLEPLAY] [99% LAG FREE])

Reporting client version: 513.1524

This is on Ice Box but it's apparently happening on all maps according to Pumpkin0.

![image](https://user-images.githubusercontent.com/7019927/82793387-6cdbfb00-9e36-11ea-9d43-941dcd992261.png)

I'm specifically referring to the tiles on the edge with decals.

**Comment by LemonInTheDark** [comment]
did you get the icon state?

**Comment by TheVekter** [comment]
Yeah, it's on floors.dmi just using the floor for the icon state. There are two overlays but they're both set to null.

**Comment by Pumpkinoe** [comment]
Did this get fixed? I haven't seen it happen in a while.

**Comment by Fikou** [comment]
doesnt happen anymore, except for the nukie shuttle for some reason??
![image](https://user-images.githubusercontent.com/23585223/90394898-ccdbe980-e093-11ea-9043-b4df59887833.png)
